generated/nimxlcterminaladaptor_restricted/nimxlcterminaladaptor_restricted.proto

```diff
@@ -0,0 +1,101 @@
+
+//---------------------------------------------------------------------
+// This file is automatically generated. All manual edits will be lost.
+//---------------------------------------------------------------------
+// Proto file for the NI-MXLCTERMINALADAPTOR-RESTRICTED Metadata
+//---------------------------------------------------------------------
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.ni.grpc.nimxlcterminaladaptorrestricted";
+option java_outer_classname = "NimxlcTerminalAdaptorRestricted";
+option csharp_namespace = "NationalInstruments.Grpc.NimxlcTerminalAdaptorRestricted";
+
+package nimxlcterminaladaptor_restricted_grpc;
+
+import "session.proto";
+
+service NimxlcTerminalAdaptorRestricted {
+  rpc CreateSession(CreateSessionRequest) returns (CreateSessionResponse);
+  rpc DestroySession(DestroySessionRequest) returns (DestroySessionResponse);
+  rpc RefreshTerminalCache(RefreshTerminalCacheRequest) returns (RefreshTerminalCacheResponse);
+  rpc HasTerminalInformationChanged(HasTerminalInformationChangedRequest) returns (HasTerminalInformationChangedResponse);
+  rpc GetSystemChangeNumber(GetSystemChangeNumberRequest) returns (GetSystemChangeNumberResponse);
+  rpc GetDeviceContainer(GetDeviceContainerRequest) returns (GetDeviceContainerResponse);
+}
+
+message NIErrStatus {
+  int32 code = 1;
+  uint32 capacity = 2;
+  string json = 3;
+}
+
+message NIMXLCDevice {
+  string name = 1;
+  bool supportsOnboardClock = 2;
+  repeated NIMXLCTerminal terminals= 3;
+}
+
+message NIMXLCTerminal {
+  string name = 1;
+  string visibility = 2;
+}
+message CreateSessionRequest {
+  string session_name = 1;
+  string hostname = 2;
+}
+
+message CreateSessionResponse {
+  int32 status = 1;
+  NIErrStatus c_status = 2;
+  nidevice_grpc.Session handle = 3;
+}
+
+message DestroySessionRequest {
+  nidevice_grpc.Session session = 1;
+}
+
+message DestroySessionResponse {
+  int32 status = 1;
+}
+
+message RefreshTerminalCacheRequest {
+  nidevice_grpc.Session session = 1;
+}
+
+message RefreshTerminalCacheResponse {
+  int32 status = 1;
+  NIErrStatus c_status = 2;
```

Comment:
Will `NIErrStatus` be needed in the response when this information is incorporated into the `status` and `trailing_metadata`?